Say Their Names/Source/Controller/Home/HomeController.swift

```diff
@@ -50,13 +51,42 @@ class HomeController: BaseViewController {
     }
     
     fileprivate func setupCollectionView() {
+        
+        // TO-DO: Dummy data for now, should update after API call to get locations
+        let locations: [Location] = [.init(name: "ALL"),
+                         .init(name: "RECENT"),
+                         .init(name: "MISSOURI"),
+                         .init(name: "TEXAS"),
+                         .init(name: "NEW YORK")]
+        
+        var people: [Person] = []
+        let df = DateFormatter()
```

Comment:
I strongly agree with this. We should have a `DateFormatterService` since `DateFormatter` is an expensive object. This task is outside the scope of this commit, so I will make a new ticket in Trello for it.